spec/integration/api/photos_controller_spec.rb

```diff
@@ -0,0 +1,293 @@
+# frozen_sTring_literal: true
+
+require "spec_helper"
+
+describe Api::V1::PhotosController do
+  let(:auth) { FactoryGirl.create(:auth_with_read_and_write) }
+  let(:auth_read_only) { FactoryGirl.create(:auth_with_read) }
+  let!(:access_token) { auth.create_access_token.to_s }
+  let!(:access_token_read_only) { auth_read_only.create_access_token.to_s }
+
+  before do
+    alice_private_spec = alice.aspects.create(name: "private aspect")
+    alice.share_with(eve.person, alice_private_spec)
+    @private_photo1 = alice.post(:photo, pending: false, user_file: File.open(photo_fixture_name),
+                                to: alice_private_spec.id)
+    @alice_public_photo = alice.post(:photo, pending: false, user_file: File.open(photo_fixture_name), public: true)
+    @user_photo1 = auth.user.post(:photo, pending: true, user_file: File.open(photo_fixture_name), to: "all")
+    @user_photo2 = auth.user.post(:photo, pending: true, user_file: File.open(photo_fixture_name), to: "all")
+    message_data = {status_message: {text: "Post with photos"}, public: true, photos: [@user_photo2.id.to_s]}
+    @status_message = StatusMessageCreationService.new(auth.user).create(message_data)
+    @user_photo2.reload
+  end
+
+  describe "#show" do
+    context "succeeds" do
+      it "with correct GUID user's photo and access token" do
+        get(
+          api_v1_photo_path(@user_photo1.guid),
+          params: {access_token: access_token}
+        )
+        expect(response.status).to eq(200)
+        photo = JSON.parse(response.body)
+        expect(photo.has_key?("post")).to be_falsey
+        confirm_photo_format(photo, @user_photo1, auth.user)
+      end
+
+      it "with correct GUID user's photo used in post and access token" do
```

Comment:
The test description doesn't seem to match what is tested here.

Reply:
Yep I forgot to use photo2 which was added to the post and then change the test of the post GUID.  Let me fix that...